<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GerberPlotterDialog</class>
 <widget class="QDialog" name="GerberPlotterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10">
   <item>
    <widget class="QGroupBox" name="propertiesGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Output File Properties</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QGroupBox" name="fileFormatGroupBox">
        <property name="title">
         <string>File Format</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_imageFormats">
           <item>
            <widget class="QRadioButton" name="radioButton_jpeg">
             <property name="text">
              <string>JPEG</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_png">
             <property name="text">
              <string>PNG</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_imageFormatProperties">
           <item>
            <widget class="QGroupBox" name="compressionQualityGroupBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Quality</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QSpinBox" name="jpgQualityBox">
                <property name="suffix">
                 <string>%</string>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="value">
                 <number>100</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="backgroundColorGroupBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Background Color</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QComboBox" name="bgColorComboBox">
                <item>
                 <property name="text">
                  <string>transparent</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>black</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>white</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>custom...</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="imageSizeGroupBox">
        <property name="title">
         <string>Image Size</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_imageSizeLabel">
           <item>
            <widget class="QLabel" name="imageWidthLabel">
             <property name="text">
              <string>Width</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="imageTimesLabel">
             <property name="maximumSize">
              <size>
               <width>18</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string notr="true">x</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="imageHeightLabel">
             <property name="text">
              <string>Height</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_imageSize">
           <item>
            <widget class="QSpinBox" name="imgWidthBox">
             <property name="suffix">
              <string> px</string>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>99999</number>
             </property>
             <property name="value">
              <number>1600</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="imgTimesLabel">
             <property name="maximumSize">
              <size>
               <width>18</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string notr="true">x</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="imgHeightBox">
             <property name="readOnly">
              <bool>false</bool>
             </property>
             <property name="suffix">
              <string> px</string>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>99999</number>
             </property>
             <property name="value">
              <number>800</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="flipBottomCheckBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Flip bottom side horizontally</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="detailsScrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>542</width>
        <height>635</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="layerGroupBox">
         <property name="title">
          <string>Layer</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QGroupBox" name="boardOutlineGroupBox">
            <property name="title">
             <string>Board Outline (Border)</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QLineEdit" name="boardOutlineLineEdit"/>
             </item>
             <item>
              <widget class="QPushButton" name="boardOutlineBrowseButton">
               <property name="text">
                <string>Browse...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="drillsGroupBox">
            <property name="title">
             <string>Drills / Holes</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <widget class="QLineEdit" name="drillsLineEdit">
               <property name="readOnly">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="drillsBrowseButton">
               <property name="text">
                <string>Browse...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <property name="elideMode">
             <enum>Qt::ElideNone</enum>
            </property>
            <widget class="QWidget" name="tabTop">
             <attribute name="title">
              <string>Top</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QGroupBox" name="topCopperGroupBox">
                <property name="title">
                 <string>Copper</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QLineEdit" name="topCopperLineEdit"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="topCopperBrowseButton">
                   <property name="text">
                    <string>Browse...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="topSolderMaskGroupBox">
                <property name="title">
                 <string>Solder Mask</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <widget class="QLineEdit" name="topSolderMaskLineEdit"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="topSolderMaskBrowseButton">
                   <property name="text">
                    <string>Browse...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="topSilkscreenGroupBox">
                <property name="title">
                 <string>Silkscreen</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <widget class="QLineEdit" name="topSilkscreenLineEdit"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="topSilkscreenBrowseButton">
                   <property name="text">
                    <string>Browse...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tabBottom">
             <attribute name="title">
              <string>Bottom</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QGroupBox" name="bottomCopperGroupBox">
                <property name="title">
                 <string>Copper</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_10">
                 <item>
                  <widget class="QLineEdit" name="bottomCopperLineEdit"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="bottomCopperBrowseButton">
                   <property name="text">
                    <string>Browse...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="bottomSolderMaskGroupBox">
                <property name="title">
                 <string>Solder Mask</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_9">
                 <item>
                  <widget class="QLineEdit" name="bottomSolderMaskLineEdit"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="bottomSolderMaskBrowseButton">
                   <property name="text">
                    <string>Browse...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="bottomSilkscreenGroupBox">
                <property name="title">
                 <string>Silkscreen</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_11">
                 <item>
                  <widget class="QLineEdit" name="bottomSilkscreenLineEdit"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="bottomSilkscreenBrowseButton">
                   <property name="text">
                    <string>Browse...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="colorSchemeGroupBox">
         <property name="title">
          <string>Color Scheme</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_colorSchemes">
            <item>
             <widget class="QRadioButton" name="radioButton_green">
              <property name="text">
               <string>green</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_blue">
              <property name="text">
               <string>blue</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_red">
              <property name="text">
               <string>red</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_yellow">
              <property name="text">
               <string>yellow</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_white">
              <property name="text">
               <string>white</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_black">
              <property name="text">
               <string>black</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_custom">
              <property name="text">
               <string>custom</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_colorPalettes">
            <item>
             <widget class="QGroupBox" name="baseColorGroupBox">
              <property name="title">
               <string>Base</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_19">
               <item>
                <widget class="QLineEdit" name="baseColorLineEdit"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="copperColorGroupBox">
              <property name="title">
               <string>Copper</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_20">
               <item>
                <widget class="QLineEdit" name="copperColorLineEdit"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="silkscreenColorGroupBox">
              <property name="title">
               <string>Silkscreen</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_21">
               <item>
                <widget class="QLineEdit" name="silkscreenColorLineEdit"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="padsColorGroupBox">
              <property name="title">
               <string>Pads</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_22">
               <item>
                <widget class="QComboBox" name="padsColorComboBox">
                 <item>
                  <property name="text">
                   <string>gold</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>silver</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>custom...</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="bumpMapGroupBox">
              <property name="title">
               <string>Bump Map</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_23">
               <item>
                <widget class="QSpinBox" name="bumpMapSpinBox">
                 <property name="suffix">
                  <string>%</string>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="value">
                  <number>50</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="outputFilesGroupBox">
     <property name="title">
      <string>Output Files</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QGroupBox" name="topSideOutputGroupBox">
        <property name="title">
         <string>Top Side</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="topSideLineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="topSideBrowseButton">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="bottomSideOutputGroupBox">
        <property name="title">
         <string>Bottom Side</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLineEdit" name="bottomSideLineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="bottomSideBrowseButton">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_Buttons">
     <property name="topMargin">
      <number>4</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_betweenButtons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="evalBoardSizeButton">
       <property name="text">
        <string>Get Size from Outline</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="generateButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Generate</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
